cocoa/src/toga_cocoa/container.py

```diff
@@ -0,0 +1,128 @@
+from rubicon.objc import objc_method
+
+from .libs import (
+    NSLayoutAttributeBottom,
+    NSLayoutAttributeLeft,
+    NSLayoutAttributeRight,
+    NSLayoutAttributeTop,
+    NSLayoutConstraint,
+    NSLayoutRelationGreaterThanOrEqual,
+    NSView,
+)
+
+
+class TogaView(NSView):
+    @objc_method
+    def isFlipped(self) -> bool:
+        # Default Cocoa coordinate frame is around the wrong way.
+        return True
+
+
+class Container:
+    def __init__(
+        self,
+        min_width=100,
+        min_height=100,
+        layout_native=None,
+        on_refresh=None,
+    ):
+        """A container for layouts.
+
+        Creates and enforces minimum size constraints on the container widget.
+
+        :param min_width: The minimum width to enforce on the container
+        :param min_height: The minimum height to enforce on the container
+        :param layout_native: The native widget that should be used to provide size
+            hints to the layout. By default, this will usually be the container widget
+            itself; however, for widgets like ScrollContainer where the layout needs to
+            be computed based on a different size to what will be rendered, the source
+            of the size can be different.
+        :param on_refresh: The callback to be notified when this container's layout is
+            refreshed.
+        """
+        self._content = None
+        self.on_refresh = on_refresh
+
+        # macOS always renders at 96dpi. Scaling is handled
+        # transparently at the level of the screen compositor.
+        self.dpi = 96
+        self.baseline_dpi = self.dpi
```

Comment:
Can these properties now be removed from every backend except Android and Winforms?

Reply:
I believe so. I originally retained them because I figured it couldn't hurt to track DPI, even if it *was* a constant; but they're not actually being used anywhere, so we could just include a comment/docstring on container that provides the detail.